pages/_error.tsx

```diff
@@ -0,0 +1,52 @@
+// CSS
+import styles from '../styles/Error.module.scss';
+// Components
+import NavBar from '../components/NavBar';
+import Button, { ButtonType } from '../components/Button';
+
+export default function ErrorPage({ 
+  statusCode 
+}:{
+  statusCode: any
+}) {
+  const returnTo = '/';
+
+  return (
+    <div className={styles.canvas}>
+      <NavBar currentPage='Home' />
+      <div className={styles.page}>
+        <div className={styles.container}>
+          <h1 className={styles.heading}>
+          Oops!
+          </h1>
+          <p className={styles.body}>
+            {statusCode
+              ? `A ${statusCode} error occurred on server`
+              : 'An error occurred on client'}
```

Comment:
I think having text similar to your first draft rather than saying that "a 404 error occurred on the server". Something that tells a non tech savvy person that the page doesn't exist or is being developed

Reply:
Are we thinking the first draft from the description (longer and personable) or the last draft (sort and simpler)?